donations.md

```diff
@@ -16,8 +16,23 @@ You can make a donation easily through Paypal, by clicking on the 'Donate' butto
 
 We thank the following individuals and companies for their donations to the NUnit project.
 
+### 2016
+
+- MacFarlane Physiotherapy, [www.macfarlanephysio.co.uk](http://www.macfarlanephysio.co.uk)
+- Dzmitry Lahoda
+- Righthand Miha Markic s.p.
+- Erl Egestad
+
```

Comment:
@CharliePoole I will create an issue to add the new donors. Once you have their names, I can do a PR.